launch/launch/actions/register_event_handler.py

```diff
@@ -32,10 +32,11 @@ class RegisterEventHandler(Action):
     place.
     """
 
-    def __init__(self, event_handler: EventHandler, **kwargs) -> None:
+    def __init__(self, event_handler: EventHandler, handle_once: bool = False, **kwargs) -> None:
```

Comment:
to force that the caller specifies the `handle_once` argument by name, please add `*, ` before it (here's an example: https://github.com/ros2/launch/blob/b6085818b723944dbf8f49eae75251fe30b0ab34/launch_ros/launch_ros/actions/lifecycle_node.py#L40)

Reply:
👍